app/views/pages/home.html.erb

```diff
@@ -1 +1,44 @@
-<h2>Welcome to Classroom!</h2>
+<div class="signin-section">
+  <div class="container text-center">
+    <ul>
+      <li><%= mega_octicon('book')%></li>
+      <li><%= mega_octicon('pencil')%></li>
+      <li><%= mega_octicon('mortar-board')%></li>
+      <li><%= mega_octicon('organization')%></li>
+      <li><%= mega_octicon('calendar')%></li>
+      <li><%= mega_octicon('comment-discussion')%></li>
+    </ul>
+
+    <h1 class="heading">Your course assignments on GitHub</h1>
+    <p class="subheading">Automate repository creation and access control, making it easy to distribute starter code and collect assignment submissions on GitHub.</p>
+
+    <h2><%= link_to 'Sign in with your GitHub account to get started', login_path %></h2>
+  </div>
+</div>
+
+<div class="information-section">
+  <div class="container">
+    <div class="columns">
+      <div class="one-third column">
+        <h2>Create your assignments</h2>
+        <p>
+          You have the ability to create both individual or group assignments, that can be public or private on GitHub.
+        </p>
+      </div>
+
+      <div class="one-third column">
+        <h2>Invite your students</h2>
+        <p>
+          Each assigment comes with a unique invitation url, so adding students to an organization is much easier.
```

Comment:
Students gain access to assignments using unique invitation URLs, eliminating the need to collect GitHub usernames in advance and manage access manually.